processor/spanmetricsprocessor/processor.go

```diff
@@ -228,28 +229,33 @@ func (p *processorImp) ConsumeTraces(ctx context.Context, traces ptrace.Traces)
 	// that should not interfere with the flow of trace data because
 	// it is an orthogonal concern to the trace flow (it should not impact
 	// upstream or downstream pipeline trace components).
```

Comment:
Parts of this comment are no longer accurate. Can you update it to reflect the new process?

Reply:
Thanks, updated comment in: [af3dc8f](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/12661/commits/af3dc8fb56a23be29095185302ea487a4e792de8). 

It's mostly deleting since I don't think that line of code is too controversial.